PeakLocalisations/Localisation/Core/Stats.xcstrings

```diff
@@ -345,6 +459,12 @@
             "state" : "translated",
             "value" : "Workout Stat"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Estadística de Entreno"
```

Comment:
Entrenamiento instead of Entreno, here and elsewhere

Reply:
Apple seems to be using "Entreno" for Spain and "Entrenamiento" for Latin America, so I just matched them here https://applelocalization.com/?q=Workout&b=HealthKit.framework&l=English&l=Spanish

Reply:
Oooh ok I never head of entreno.